packages/website/content/pages/app/account.json

```diff
@@ -76,7 +76,8 @@
       },
       "loading": "Loading...",
       "max_space_tib_label": "TiB",
-      "unlock_label": "1 TiB+"
+      "unlock_label": "1 TiB+",
+      "tooltip_total": "<div> This is the total of all your uploads, including ones marked \"Failed\" (you can read more about what the status means <a href='https://web3.storage/docs/how-tos/get-status/'>here</a>). </div> You can remove uploads via the table below, through your preferred client library, the CLI or the HTTP APIs.<div>"
```

Comment:
What's the purpose of the trailing `<div>` on the end of this?

Reply:
I updated the copy and missed that closing tag. 
Great catch

Reply:
👍 fix.